ui/app/templates/components/clients/monthly-usage.hbs

```diff
@@ -1,12 +1,12 @@
-<div class="chart-wrapper single-chart-grid">
+<div class="chart-wrapper single-chart-grid" data-test-monthly-usage>
```

Comment:
Same here: this is on this pr but not the original?

Reply:
Original PR for comparison:
<img width="474" alt="Screen Shot 2022-06-06 at 10 40 09 AM" src="https://user-images.githubusercontent.com/68122737/172215394-8a07e4e5-8582-4ee3-80c8-92064ea11a88.png">